packages/card/src/Card.js

```diff
@@ -43,6 +53,22 @@ const Root = createStyledComponent(
 /**
  * Card component
  */
-export default function Card(props: Props) {
-  return <Root {...props} />;
+export default class Card extends Component {
+  props: Props;
+
+  render() {
+    const rootProps = {
+      onKeyPress: this.props.onClick ? this._onKeyPress.bind(this) : undefined,
+      tabIndex: this.props.onClick ? 0 : undefined,
+      ...this.props
+    };
+    return <Root {...rootProps} />;
+  }
+
+  _onKeyPress(event: Object) {
+    if (this.props.onClick && (event.key === 'Enter' || event.key === ' ')) {
+      event.preventDefault();
+      this.props.onClick(event);
```

Comment:
Flow says `Function cannot be called on possibly undefined value` on this line, but I'm checking if it exists in the `if` statement. Any ideas?